packages/playground-examples/copy/ja/3-8/Syntax and Messaging/Type Imports.tsx

```diff
@@ -0,0 +1,48 @@
+//// { compiler: { ts: "3.8.3" } }
+// 3.8では、型をインポートするための新しい構文を追加しました。
+// これはflowユーザーにはおなじみのものです。
+
+// 'import type'は、型のみのインポートを宣言する方法を提供します。これは、
+// JavaScriptに変換される際に必ずコードが削除されるため、
+// どのコードが削除されるのか非常に予測しやすいということを意味します！
```

Comment:
かなり訳に困ったので、より良い訳募集します
```suggestion
// 'import type'は、型のみのインポートを宣言する方法を提供します。
// これは、とても予測可能な方法でJavaScriptに変換される際に
// そのコードが削除されると確信できることを意味します。
// なぜなら、型は必ず削除されるからです！
```

Reply:
なるほど！itは型なのか...
十分わかりやすいと思います 👍 
```
// 'import type' provides a way to declare a type only import
// which means you can be sure that the code will erased when
// converting to JavaScript in a very predictable way because
// it will always be removed!
```

Reply:
では、suggestのやつで行きましょう